temporal-sdk/src/test/java/io/temporal/workflow/nexus/SyncOperationCancelledTest.java

```diff
@@ -58,17 +58,6 @@ public void syncOperationImmediatelyCancelled() {
         "operation canceled before it was started", canceledFailure.getOriginalMessage());
   }
 
-  @Test
-  public void syncOperationCancelled() {
```

Comment:
Removed this test since the Server dropped support for cancelling failing sync operations